src/context-providers/display/display-provider.tsx

```diff
@@ -0,0 +1,63 @@
+import noop from "lodash/noop";
+import { createContext } from "react";
+import { useImmerReducer } from "use-immer";
+import { IDisplayState, TDisplayAction, TDisplayContext } from "./types";
+
+// =============================================================================
+// DEFAULT VALUES
+// =============================================================================
+const DEFAULT_DISPLAY_VALUES: IDisplayState = {
+    toast: [],
```

Comment:
apologies may I suggest to name it in plural form because there can be more than 1 toast

```suggestion
    toasts: [],
```

Reply:
changes are made on the latest push